big_tests/tests/amp_big_SUITE.erl

```diff
@@ -85,39 +92,48 @@ basic_test_cases() ->
 
      last_rule_applies_test].
 
-notify_deliver_test_cases() ->
+deliver_test_cases(notify) ->
     [notify_deliver_to_online_user_test,
      notify_deliver_to_online_user_bare_jid_test,
      notify_deliver_to_online_user_recipient_privacy_test,
      notify_deliver_to_offline_user_test,
      notify_deliver_to_offline_user_recipient_privacy_test,
      notify_deliver_to_stranger_test,
-     notify_deliver_to_malformed_jid_test].
-
-error_deliver_test_cases() ->
+     notify_deliver_to_malformed_jid_test];
+deliver_test_cases(error) ->
     [error_deliver_to_online_user_test,
      error_deliver_to_offline_user_test,
-     error_deliver_to_stranger_test].
-
-drop_deliver_test_cases() ->
+     error_deliver_to_stranger_test];
+deliver_test_cases(drop) ->
     [drop_deliver_to_online_user_test,
      drop_deliver_to_offline_user_test,
      drop_deliver_to_stranger_test].
 
-init_per_suite(C) ->
+%% Setup and teardown
+
+init_per_suite(Config) ->
     rpc(mim(), ejabberd_config, add_local_option, [{{s2s_host, <<"not a jid">>}, domain()}, deny]),
-    escalus:init_per_suite(C).
+    ConfigWithHooks = [{ct_hooks, [{multiple_config_cth, fun tests_with_config/1}]} | Config],
```

Comment:
How does it work with the `ct_hooks` defined in `default.spec` file? Are they still respected even if here we specify one element list?

Reply:
https://erlang.org/doc/apps/common_test/ct_hooks_chapter.html#overriding-cths

"By default, each installation of a CTH causes a new instance of it to be activated."

I even traced the tests to make sure it works :-)

Reply:
All right, thanks for the explanation!